src/index.tsx

```diff
@@ -15,6 +16,7 @@ import { getConfig } from 'config'
 import React from 'react'
 import { createRoot } from 'react-dom/client'
 import { httpClientIntegration } from 'utils/sentry/httpclient'
+import { v4 as uuid } from 'uuid'
 import { renderConsoleArt } from 'lib/consoleArt'
 import { reportWebVitals } from 'lib/reportWebVitals'
 
```

Comment:
to test locally, set the line below
`if (window.location.hostname !== 'localhost') {`
to 
`if (true) {`